scripts/openstack/manifest-tests/server-groups/install-config.yaml

```diff
@@ -10,6 +10,7 @@ controlPlane:
       serverGroupPolicy: anti-affinity
       zones:
       - zoneone
+      - zonetwo
```

Comment:
@pierreprinetti What does adding this do?

I would expect a change like this to include test changes to be input and expected output. What's the context here?

Reply:
This change adds the failing test that the rest of the code fixes.

The test says "all masters have the same server group". With this change, that test is executed with an install-config that gives masters two different AZs (rather than just one).